cms/templates/js/course_grade_policy.underscore

```diff
@@ -13,22 +13,41 @@
 
   <div class="field text" id="field-course-grading-assignment-gradeweight">
     <label for="course-grading-assignment-gradeweight"><%= gettext("Weight of Total Grade") %></label>
-    <input type="text" class="short" id="course-grading-assignment-gradeweight" value = "<%= model.get('weight') %>" />
+    <input type="text" class="short" id="course-grading-assignment-gradeweight" value="<%= model.get('weight') %>" />
     <span class="tip tip-inline"><%= gettext("as a percent, e.g. 40") %></span>
   </div>
 
   <div class="field text" id="field-course-grading-assignment-totalassignments">
     <label for="course-grading-assignment-totalassignments"><%= gettext("Total Number") %></label>
-    <input type="text" class="short" id="course-grading-assignment-totalassignments" value = "<%= model.get('min_count') %>" />
+    <input type="text" class="short" id="course-grading-assignment-totalassignments" value="<%= model.get('min_count') %>" />
     <span class="tip tip-inline"><%= gettext("total exercises assigned") %></span>
   </div>
 
   <div class="field text" id="field-course-grading-assignment-droppable">
     <label for="course-grading-assignment-droppable"><%= gettext("Number of Droppable") %></label>
-    <input type="text" class="short" id="course-grading-assignment-droppable" value = "<%= model.get('drop_count') %>" />
+    <input type="text" class="short" id="course-grading-assignment-droppable" value="<%= model.get('drop_count') %>" />
     <span class="tip tip-inline"><%= gettext("total exercises that won't be graded") %></span>
   </div>
 
+  <div class="clearfix"></div>
+  <div class="field field-checkbox checkbox-cosmetic" id="assignment-passing-grade-enabled">
+    <input type="checkbox" id="passing-grade-enabled" name="passing-grade-enabled" class="input input-checkbox">
```

Comment:
There is a UI bug in edX: They're using IDs here and it leads to id duplication that is wrong. It'd be great if you can solve it (avoid using ids in this template). But it may require extra work, especially in fixing js files and bok-choy tests.